spinalcordtoolbox/image.py

```diff
@@ -1245,9 +1245,7 @@ def zeros_like(img, dtype=None):
     intent and avoid doing a copy, which is slower than initialization with a constant.
 
     """
-    dst = change_type(img, dtype)
-    dst.data[:] = 0
-    return dst
+    return Image(np.zeros_like(img.data), hdr=img.hdr).change_type(dtype)
```

Comment:
LGTM.

Potential nitpick: if you write it this way:
```python
zimg = Image(np.zeros_like(img.data), hdr=img.hdr)
if dtype is not None:
    zimg.change_type(dtype)
return zimg
```
It makes it obvious that `change_type(dtype=None)` has no potential side-effect. Right now one would need to look into `Image.change_type()` to make sure of that.